delta/sourcing-psql/src/test/scala/ch/epfl/bluebrain/nexus/delta/sourcing/stream/pipes/SelectPredicatesSuite.scala

```diff
@@ -54,7 +54,7 @@ class SelectPredicatesSuite extends BioSuite {
     registry
       .lookupA[SelectPredicates.type](SelectPredicates.ref)
       .rightValue
-      .withJsonLdConfig(SelectPredicatesConfig(predicates).toJsonLd)
+      .withJsonLdConfig(SelectPredicatesConfig(None, predicates).toJsonLd)
```

Comment:
Should the forward type `File` be set here?

Reply:
It could, I chose not to

Reply:
Did this not indicate a problem?